operations/model.go

```diff
@@ -29,6 +29,20 @@ const stagingNonCorpHost = "https://evergreen-staging.corp.mongodb.com/api"
 const prodCorpHost = "https://evergreen.corp.mongodb.com/api"
 const prodNonCorpHost = "https://evergreen.mongodb.com/api"
 
+// RestCommunicatorOption is a functional option for configuring the REST communicator setup.
+type RestCommunicatorOption func(*restCommunicatorOptions)
```

Comment:
This + `SkipCheckingMinimumCLIVersion` aren't used outside of this package so can be unexported.